src/aircrafts/F16/fcs.jl

```diff
@@ -27,6 +29,11 @@ end
 # TODO: Move to FCS in models
 get_thrust(fcs::F16FCS) = get_value(fcs.cpl)
 
+# TODO: move this constants to aircraft parameters
```

Comment:
This variables where already declared in https://github.com/parthp08/FlightMechanics.jl/blob/master/src/aircrafts/F16/aerodynamics.jl#L23
and can be used because of the include order in `Aircrafts.jl`

This is confusing also for me, I don't feel sure about where to place this variables. But we should avoid redeclaring them. I would leave them where they are until we find a better code organization

Reply:
i think we should create new file for aircraft parameters and import it before all other  files in Aircrafts.jl. what do you think?

Reply:
In the line with the previous comment, I think most julia packages tend to group code in larger files. Maybe it makes more sense to have everything in the same file with the constants at the top.

Reply:
wont it create too large file. it will also be difficult add new features that way.